lib/src/chart/line_chart/line_chart_painter.dart

```diff
@@ -326,4 +348,31 @@ class LineChartPainter extends AxisChartPainter {
 
   @override
   bool shouldRepaint(CustomPainter oldDelegate) => false;
+
+  void drawExtraLines(Canvas canvas, Size viewSize) {
```

Comment:
Here plz draw both horizontal and vertical lines, (firstly data class should be updated)

Reply:
The drawing of the two line types has been merged into this method.

Reply:
But I'm not sure what you mean with that the data class should be updated?